libcxx/docs/ReleaseNotes/21.rst

```diff
@@ -52,6 +52,8 @@ Improvements and New Features
 
 - Updated formatting library to Unicode 16.0.0.
 
+- ``std::erase`` and ``std::erase_if`` overloads for ``std::basic_string`` are made ``constexpr`` in C++20 and later.
```

Comment:
I'm not sure this is worth a release note. It's on the same kind of level as an LWG issue, which we don't release note either.

Reply:
Thanks! I've reverted the changes in release notes.